googleanalytics.module

```diff
@@ -386,9 +386,9 @@ function googleanalytics_preprocess_layout() {
       // @FIXME: Cannot find the debug URL!???
       $library = 'https://www.googletagmanager.com/gtag/js?id=' . $id_list[0];
     }
-    elseif (variable_get('googleanalytics_cache', 0) && $url = _googleanalytics_cache('https://www.googletagmanager.com/gtag/js')) {
+    elseif ($config->get('googleanalytics_cache') && $url = _googleanalytics_cache('https://www.googletagmanager.com/gtag/js')) {
```

Comment:
This key should be simply `cache` as I removed the `googleanalytics` from all the variable names. https://github.com/backdrop-contrib/googleanalytics/blob/1.x-2.x/config/googleanalytics.settings.json#L34